kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/JobServiceIntegrationTest.java

```diff
@@ -529,4 +529,38 @@ public void testScheduleAndRunJobWithWorkItem() throws Exception {
                      jobRequest.getCommandName());
         assertNotNull(jobRequest.getData().get("workItem"));
     }
+
+    @Test
+    public void testGetDeletedJob() {
+        final long jobId = 1L;
+        KieServicesException expectedException = new KieServicesException();
+
+        try {
+            jobServicesClient.getRequestById(jobId,
+                                             false,
+                                             false);
```

Comment:
I would suggest adding a "fail" right after the getRequestById call, that way the entire test would fail if the exception is not thrown as expected. You can then also just validate the exception inside the catch statement.